tests/Service/Messaging/Mqtt/CapabilitiesServiceTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
```

Comment:
Since there is a new package for MQTT tests, former HTTP based tests should be moved to a dedicated package.

Reply:
I move the tests step by step as the mqtt implementation is ready. Is that ok?

Reply:
Fine for me.